docs/source/tutorial.rst

```diff
@@ -43,14 +43,14 @@ We could fetch a list of available analysis methods and print them by executing:
 
```

Comment:
While you're at it, could you remove the comma in "There are different analysis methods, one can choose from" on line 36?